benchmarks/showcases/LazyGridImageView/download_images.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+TARGET_DIR="composeApp/src/commonMain/composeResources/drawable"
+TARGET_DIR_IOS="nativeiosApp/nativeiosApp/Assets.xcassets"
+TRAGET_DIR_ANDROID="nativeAndroidApp/app/src/main/assets/drawable"
+
+mkdir -p "$TARGET_DIR"
+mkdir -p "$TARGET_DIR_IOS"
+mkdir -p "$TARGET_DIR_ANDROID"
+
+echo "Downloading 999 images from picsum.photos..."
+
+# Download 999 images
+for i in $(seq 1 999); do
+    imagename="image$(printf "%03d" $i)" 
+    filename="${imagename}.jpg"
+    filepath="$TARGET_DIR/$filename"
+
+    echo "Downloading image $i/999: $filename"
+
+    # Use a specific image ID to ensure we get different images
+    # picsum.photos has images with IDs from 0 to about 1084
+    # We'll use modulo to cycle through available IDs if we need more than available
+    image_id=$((i % 1085))
+
+    # Download the image with specific ID, size 512x512, in JPG format
+    curl -s -L -o "$filepath" "https://picsum.photos/id/$image_id/512/512.jpg"
```

Comment:
```suggestion
    curl -f -s -L -o "$filepath" "https://picsum.photos/id/$image_id/512/512.jpg"
```
(checked it on Windows)

Without it, it downloads https://picsum.photos/id/968/512/512.jpg (and 86 other images) on my machine, even if it returns 404

Reply:
thanx, fixed